README.md

```diff
@@ -20,10 +20,11 @@ To install the latest (stable) development version instead:
 
 ```julia
 using Pkg
+Pkg.add(PackageSpec(url="https://github.com/plotly/DashBase.jl.git"))
+Pkg.add(PackageSpec(url="https://github.com/waralex/dash-html-components.git", rev="jl_generated"))
+Pkg.add(PackageSpec(url="https://github.com/waralex/dash-core-components.git", rev="jl_generated"))
+Pkg.add(PackageSpec(url="https://github.com/waralex/dash-table.git", rev="jl_generated"))
```

Comment:
These (and in `ci_prepare.jl`) can point to plotly, rev="master" now I guess?

Reply:
If we have generated components in master, then yes

Reply:
Yep, if you're happy with eg https://github.com/plotly/dash-core-components/tree/master then let's do it!

Reply:
@alexcjohnson done in 1cfaf4a